identity-apps-core/apps/recovery-portal/src/main/webapp/username-recovery.jsp

```diff
@@ -200,24 +200,27 @@
                 <div class="segment-form">
                     <form class="ui large form" method="post" action="verify.do" id="recoverDetailsForm">
                         <% if (isFirstNameInClaims || isLastNameInClaims) { %>
-                        <div class="field">
-                            <label><%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "name")%></label>
-                            <div class="two fields">
-                                <% if (isFirstNameInClaims) { %>
-                                <div class="required field">
-                                    <input id="first-name" type="text" required name="http://wso2.org/claims/givenname"
-                                        placeholder="<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle,
-                                            "First.name")%>*" />
-                                </div>
-                                <% } %>
-                                <% if (isLastNameInClaims) { %>
-                                <div class="field">
-                                    <input id="last-name" type="text" name="http://wso2.org/claims/lastname"
-                                        placeholder="<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle,
-                                            "Last.name")%>" />
-                                </div>
-                                <% } %>
+                        <div class="two fields">
+                            <% if (isFirstNameInClaims) { %>
+                            <div class="required field">
+                                <label>
+                                    <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "First.name")%>
```

Comment:
Let's use `i18n` method in localize.jsp instead/

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7502/commits/82052abeac4e20e0c6c313fd8b2a8580c966a531